rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/domain/PersonalData.java

```diff
@@ -0,0 +1,66 @@
+package pl.oaza.warszawa.dor.rekolekcje.api.participants.domain;
+
+import java.time.LocalDateTime;
+import javax.persistence.Embeddable;
+import javax.persistence.Embedded;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
```

Comment:
\[Checkstyle\] WARNING: Import statement for 'lombok\.Builder' is in the wrong order\. Should be in the 'THIRD\_PARTY\_PACKAGE' group, expecting not assigned imports on this line\.